config/config.go

```diff
@@ -71,11 +71,12 @@ func (config *Config) parseFlags() {
 	flag.DurationVar(&config.Consul.Timeout, "consul-timeout", 3*time.Second, "Time limit for requests made by the Consul HTTP client. A Timeout of zero means no timeout")
 	flag.Uint32Var(&config.Consul.AgentFailuresTolerance, "consul-max-agent-failures", 3, "Max number of consecutive request failures for agent before removal from cache")
 	flag.Uint32Var(&config.Consul.RequestRetries, "consul-get-services-retry", 3, "Number of retries on failure when performing requests to Consul. Each retry uses different cached agent")
+	flag.StringVar(&config.Consul.ConsulNameSeparator, "consul-name-separator", ".", "Separator used to create default service name for Consul")
 
 	// Web
-	flag.StringVar(&config.Web.Listen, "listen", ":4000", "Accept connections at this address")
-	flag.IntVar(&config.Web.QueueSize, "events-queue-size", 1000, "Size of events queue")
-	flag.IntVar(&config.Web.WorkersCount, "workers-pool-size", 10, "Number of concurrent workers processing events")
+	flag.StringVar(&config.Web.Listen, "listen", ":4000", "accept connections at this address")
+	flag.IntVar(&config.Web.QueueSize, "events-queue-size", 1000, "size of events queue")
+	flag.IntVar(&config.Web.WorkersCount, "workers-pool-size", 10, "number of concurrent workers processing events")
```

Comment:
Why change it to lowercase? In [last commit](https://github.com/allegro/marathon-consul/commit/6ed7737d4e5ddf82bd078005b8caa6a6b18270b5/config/config.go) I make it uppercase to be consistent with other.